README.md

```diff
@@ -101,7 +101,33 @@ would in a Django shell.
 
 Unlike our other web applications this project is not using tox. To run tests locally, run this command instead:
 
+    ### PYTHON TESTS/LINTING
+    # Run Python tests
     docker-compose run --rm web pytest
+    # Run Python tests in a single file
+    docker-compose run --rm web pytest /path/to/test.py
+    # Run Python test cases in a single file that match some function/class name
+    docker-compose run --rm web pytest /path/to/test.py -- -k test_some_logic
+    # Run Python linter
+    docker-compose run --rm web pylint
+    
+    ### JS/CSS TESTS/LINTING
+    # We also include a helper script to execute JS tests in most of our projects 
+    # (this file may exist at the project root or in ./scripts/test)
```

Comment:
No need for this line, as far as the `mitodl/mitxpro` repo is concerned the file will be found in `scripts/test/` which you have already updated in the commands below.